### ISSUE TITLE
Support for multi-project setups

### ISSUE THREAD

**Issue by schickling** [graphcool/graphql-cli#48]
`graphql-cli` doesn't seem to support multi project configurations like this:

```json
{
  "projects": {
    "database": {
      "schemaPath": "database.graphql",
      "extensions": {
        "endpoints": {
          "default": {
            "url": "https://api.graph.cool/simple/v1/xxx",
            "subscription": "wss://api.graph.cool/simple/v1/xxx"
          }
        }
      }
    },
    "gateway": {
      "schemaPath": "gateway.graphql",
      "extensions": {
        "endpoints": {
          "dev": {
            "url": "http://localhost:4000",
            "subscription": "ws://localhost:4000"
          }
        }
      }
    }
  }
}
```

Error message:

![image](https://user-images.githubusercontent.com/1567498/33014089-668dc2e2-cde6-11e7-8a0c-2442aeb756e8.png)

It should be possible to provide the project name via a `--project` argument.

**Comment by schickling** [comment]
Update: As a workaround you can set the `GRAPHQL_CONFIG_PROJECT` to your desired project.

**Comment by schickling** [comment]
Fixed in `2.0.0` release.
